common/sphinx.c

```diff
@@ -166,7 +165,7 @@ static bool generate_header_padding(
 	)
 {
 	int i;
-	u8 cipher_stream[(NUM_MAX_HOPS + 1) * hopsize];
+	u8 cipher_stream[(NUM_MAX_FRAMES + 1) * FRAME_SIZE];
```

Comment:
Either this change is wrong, or hopsize should not be a parameter to this function?